test/NuGet.Tests.Apex/NuGet.PackageManagement.UI.TestContract/ApexTestUIProject.cs

```diff
@@ -125,6 +125,14 @@ public bool WaitForSearchComplete(Action search, TimeSpan timeout)
             }
         }
 
+        /// <summary>
+        /// Used for package namespace Apex tests which require All option in package sources.
+        /// </summary>
+        public void SetPackageSourceOptionToAll() => UIInvoke(() => {
```

Comment:
Apex contract already has this pattern which `UIInvoke` switch to main UI thread with JTF run. All other methods already use `UIInvoke`, without using it throws below exception. 
`System.Runtime.Serialization.SerializationException : Type 'System.Threading.Tasks.Task`1[[System.Threading.Tasks.VoidTaskResult, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]' in Assembly 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' is not marked as serializable.`